### PR TITLE
VFB-181: Remove duplicate postcodes from Google Map generation

### PR DESCRIPTION
**Pull request by maxjohns0n**
## What's changed
* Removed duplicate postcodes
* Appended `//` to URL to avoid generating route

(Unable to reproduce the bug where the generation fails the first time.)

## Screenshots / Videos
| Before     | After      |
|------------|------------|
| image | image |

## Checklist
- [x] The ticket is up-to-date - Please document any deviations from the original approach if there is any.
- [x] I have documented the testing steps for QA
- [x] I have self-reviewed this PR
- [x] Make sure you've verified it works via `npm run dev`
- [x] Make sure you've verified it works via `npm run build` and `npm run start`
- [x] Make sure you've fixed all linting problems with `npm run lint_fix`
- [x] Make sure you've tested via `npm run test:e2e`

no db changes

### CODE REVIEW

File: src/app/parcels/ActionBar/ActionModals/GenerateMapModal.tsx
Comment (minalui) [inline src/app/parcels/ActionBar/ActionModals/GenerateMapModal.tsx:30]: Is there any way of making this more readable?

Comment (maxjohns0n) [reply src/app/parcels/ActionBar/ActionModals/GenerateMapModal.tsx]: I've moved the uniqueness finding into separate statement to make it clearer what's going on. The formatting code is now unchanged! Lmk if there are any other improvements you can think of :)

Comment (minalui) [reply src/app/parcels/ActionBar/ActionModals/GenerateMapModal.tsx]: Looks great! Let's wait for someone else to review before merging 👍

**Review by rickaw — APPROVED** [review]

**Review by 8kristy — APPROVED** [review]

**Pull request merged**
